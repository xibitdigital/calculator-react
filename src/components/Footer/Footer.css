.footer {
    background-color: #1795d4;
    color: white;
    padding: 10px;
    text-align: right;
}
