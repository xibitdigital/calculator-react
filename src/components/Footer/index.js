import React from 'react'
import './Footer.css'

const Footer = () => <footer className="footer">Â© 2018</footer>

export default Footer
