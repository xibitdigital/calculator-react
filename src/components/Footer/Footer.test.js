import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import Footer from '.'

test('should render a Footer', () => {
    const footer = shallow(<Footer />)
    expect(shallowToJson(footer)).toMatchSnapshot()
})
