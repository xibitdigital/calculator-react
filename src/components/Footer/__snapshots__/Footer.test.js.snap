// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a Footer 1`] = `
<footer
  className="footer"
>
  Â© 2018
</footer>
`;
